projects/ion/src/lib/select/select.component.ts

```diff
@@ -20,13 +20,16 @@ export class IonSelectComponent implements OnInit {
   @Input() placeholder = '';
   @Input() options: IonSelectProps['options'] = [];
   @Input() maxSelected?: IonSelectProps['maxSelected'];
+  @Input() required: IonSelectProps['required'] = false;
   @Output() events = new EventEmitter<IonSelectProps['options']>();
   @Output() search = new EventEmitter<string>();
 
   showDropdown = false;
   inputValue = '';
   visibleOptions: IonSelectProps['options'] = [];
   showPlaceholder = true;
+  private touched = false;
+  private isValue = false;
```

Comment:
I think hasValue would be more meaningful
```suggestion
  private hasValue = false;
```